### PR TITLE
Schema const interfaces

### PR DESCRIPTION
**Pull request by EdDev**

### CODE REVIEW

**Comment by EdDev** [comment]
Depends on #93

File: libnmstate/schema.py [hunk 1]
```diff
@@ -26,3 +26,7 @@ def load(schema_name):
 
 
 ifaces_schema = load('operational-state')
+
+
+class Const(object):
```
Comment (tyll) [inline libnmstate/schema.py]: I would prefer non-abbreviated names where possible, for example `constants` and it seems to be more natural to have them in a module than in a class on first thought. Btw. do you have plans to add all identifiers as constants?

Comment (EdDev) [reply libnmstate/schema.py]: It fits an Enum IMO, so having it under a Class feels correct to me. The subject also fits the schema module as well.
Changing the class/enum name.

Comment (EdDev) [reply libnmstate/schema.py]: I guess it should happen, not sure if that is an important task though.

**Pull request merged**